src/Humanizer/Humanizer.csproj

```diff
@@ -263,4 +264,4 @@
   <Target Name="AfterBuild">
   </Target>
   -->
-</Project>
+</Project>
```

Comment:
changes to this file should be reverted, as they only formatting and add no value to the feature.